blackbox/docs/sql/administration/user_management.txt

```diff
@@ -0,0 +1,109 @@
+analoguous `DROP USER`_ statement. These statements are database management
+statements that can only be invoked by superusers that already exist in the
+CrateDB cluster. Creating and dropping users can only be performed using
+clients that use the Postgres Protocol (such as the JDBC client), because user
+authentication is only supported by this protocol.
+
+Users cannot be backed up or restored using the ``CREATE SNAPSHOT`` and
+``RESTORE SNAPSHOT`` functionality.
+
+.. hide:
+
+    This document is not included in the doctest suite and therefore the
+    following statements are not executed.
+    The reason for that is, that the HTTP protocol does not support user
+    authentication yet.
+
+CREATE USER
+-----------
+
+To create a new user for the CrateDB database cluster use the
+:doc:`/sql/reference/create_user` SQL statement::
+
+    cr> CREATE USER user_a;
+    CREATE OK, 1 row affected (... sec)
+
+The newly created user does not have any special privileges.
+It can be used to connect to the database cluster using available
+authentication methods.
+
+The username parameter of the statement follows the principles of an identifier
+which means that it must be double-quoted if it contains special characters
+(e.g. whitespace) or if the case needs to be maintained::
+
+    cr> CREATE USER "Custom User";
+    CREATE OK, 1 row affected (... sec)
+
+If a user with the username specified in the SQL statement already exists the
+statement returns an error::
+
+    cr> CREATE USER "Custom User";
+    UserAlreadyExistsException ...
+
+DROP USER
+---------
+
+.. hide:
+
+    cr> CREATE USER user_b;
+    CREATE OK, 1 row affected (... sec)
+
+To remove an existing user from the CrateDB database cluster use the
+:doc:`/sql/reference/drop_user` SQL statement::
+
+    cr> DROP USER user_b;
+    DROP OK, 1 row affected (... sec)
+
+If a user with the username specified in the SQL statement does not exist the
+statement returns an error::
+
+    cr> DROP USER user_b;
+    UserDoesNotExistException ...
+
+It is also not possible to drop the built-in superuser ``crate``.
+
+List Users
+----------
+
+CrateDB exposes database users via the read-only ``sys.users`` system table.
+The ``sys.users`` table shows all users in the cluster which can be used for
+authentication. The initial superuser ``crate`` which is available for all
+CrateDB clusters is also part of that list.
+
+To list all existing users query that table::
+
+    cr> SELECT * FROM sys.users;
+    +-------------+-----------+
+    | name        | superuser |
+    +-------------+-----------+
+    | crate       | TRUE      |
+    | Custom User | FALSE     |
+    | user_a      | FALSE     |
+    +-------------+-----------+
+    SELECT 3 rows in set (... sec)
+
+The column ``name`` shows the unique name of the user, the column ``superuser``
+shows whether the user has superuser privileges or not.
+
+.. warning::
+
+    When the :ref:`es_api_setting` is enabled, it is possible to read the
```

Comment:
It's not possible to query `sys.users` without superuser privileges. It's also not possible if you have no user at all as in HTTP. I think i need to point out that fact more clearly. The ES API is the only way to by-pass that and to read user data without any authentication and authorisation as superuser.